src/shared/modules/dbMeta/dbMetaDuck.js

```diff
@@ -431,11 +433,26 @@ export const dbMetaEpic = (some$, store) =>
 
                   store.dispatch(update({ databases }))
 
-                  // Currently not using a db
-                  if (!getUseDb(store.getState())) {
-                    const defaultDb = databases.filter(db => db.default)
-                    if (defaultDb.length) {
-                      store.dispatch(useDb(defaultDb[0].name))
+                  if (getUseDb(store.getState())) return Rx.Observable.of(null)
+
+                  const connectTo = getActiveConnectionData(store.getState())
+                    ?.connectTo
+
+                  if (connectTo) {
+                    const wantedDb = databases.find(
+                      ({ name }) =>
+                        name.toLowerCase() === connectTo.toLowerCase()
+                    )
+                    if (wantedDb) {
+                      store.dispatch(useDb(wantedDb.name))
+                    } else {
+                      // this will show the db not found frame
+                      store.dispatch(executeSingleCommand(`:use ${connectTo}`))
```

Comment:
I thought this was pretty nice error handling of databases that weren't found. Thoughts?